src/main/java/nus/climods/ui/common/HelpWindow.java

```diff
@@ -102,24 +102,61 @@ public void setWindowDefaultSize(GuiSettings guiSettings) {
      */
     private void displayUserManual() {
         WebEngine webEngine = webView.getEngine();
+        boolean isConnectedToInternet = hasInternet();
+
+        if (isConnectedToInternet) {
+            preventRedirection(webEngine);
+            webEngine.load(USERGUIDE_URL);
+        } else {
+            loadLocalUserManual(webEngine);
+        }
+    }
+
+    private void loadLocalUserManual(WebEngine webEngine) {
+        try {
+            Stream<String> lines = Files.lines(
+                Paths.get(ClassLoader.getSystemResource("html/UserManual.html").toURI())
+            );
+            webEngine.loadContent(lines.collect(Collectors.joining("\n")));
+        } catch (URISyntaxException | IOException e) {
+            logger.warning("User Manual not found!");
+        }
+    }
 
+    private boolean hasInternet() {
```

Comment:
Just a consideration, do we want to put this as helper instead? I forsee we will be using this for our modules too

Reply:
Good idea!  I meant for this method to check for the availability of our github website only, since github can be down while the rest of the internet is up.  Perhaps I should name it better...